### PR TITLE
Exceptionhandler

### PR DESCRIPTION
**Pull request by msvinaykumar**
Autotune rest API’s should notify the client using JSON output if the API request was successfully handled or not.

### CODE REVIEW

File: src/main/java/com/autotune/analyzer/exceptions/AutoTuneExceptionHandler.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
+package com.autotune.analyzer.exceptions;
```
Comment (dinogun) [inline src/main/java/com/autotune/analyzer/exceptions/AutoTuneExceptionHandler.java:1]: Please add legal header

File: src/main/java/com/autotune/analyzer/exceptions/AutoTuneExceptionHandler.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            requestUri = "Unknown";
+        }
+        response.setContentType(JSON_CONTENT_TYPE);
+        response.setCharacterEncoding(CHARACTER_ENCODING);
+        JSONArray exceptionArray = new JSONArray();
+        JSONObject exceptionDetail = new JSONObject();
+        exceptionDetail.put("STATUS_CODE",statusCode);
+        exceptionDetail.put("REQUESTED_URI",requestUri);
+        exceptionDetail.put("SERVLET_NAME",servletName);
+        if(throwable!=null) {
```
Comment (dinogun) [inline src/main/java/com/autotune/analyzer/exceptions/AutoTuneExceptionHandler.java:66]: Please fix the formatting
```
if (x != y) {
}
```

File: src/main/java/com/autotune/analyzer/exceptions/AutoTuneExceptionHandler.java (hunk 1 above)
Comment (msvinaykumar) [reply src/main/java/com/autotune/analyzer/exceptions/AutoTuneExceptionHandler.java:16]: done

File: src/main/java/com/autotune/analyzer/exceptions/AutoTuneExceptionHandler.java (hunk 2 above)
Comment (msvinaykumar) [reply src/main/java/com/autotune/analyzer/exceptions/AutoTuneExceptionHandler.java]: done

**Review by dinogun — APPROVED** [review]
LGTM

**Review by chandrams — APPROVED** [review]
Thanks @msvinaykumar, looks good now

**Pull request merged**